src/index.js

```diff
@@ -1,50 +1,71 @@
 import './style.css';
+import localstorage from './localstorage.js';
+import userInteraction from './userInteraction.js';
 
-const todoData = [
-  {
-    description: 'Doing house chores',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Joining Microverse class',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Taking a nap',
-    completed: false,
-    index: 0,
-  },
-];
-
-todoData.forEach(() => {
-  for (let i = 0; i < todoData.length - 1; i += 1) {
-    if (todoData[i].index > todoData[i + 1].index) {
-      const temp = todoData[i];
-      todoData[i] = todoData[i + 1];
-      todoData[i + 1] = temp;
+/* eslint max-classes-per-file: ["error", 3] */
+
+class List {
+  constructor(description, index) {
+    this.description = description;
+    this.index = index;
+  }
+}
+
+const { UI } = userInteraction;
+const { StorageInLocal } = localstorage;
+const store = new StorageInLocal();
+const storeArray = store.getLists();
+
+storeArray.map(() => {
+  for (let i = 0; i < storeArray.length - 1; i += 1) {
+    if (storeArray[i].index > storeArray[i + 1].index) {
+      const temp = storeArray[i];
+      storeArray[i] = storeArray[i + 1];
+      storeArray[i + 1] = temp;
     }
   }
+  return null;
+});
+
+document.addEventListener('DOMContentLoaded', UI.displayAllLists);
+document.querySelector('#addForm').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const description = document.querySelector('.add-list').value;
+  const index = store.numberOfLists;
+  const list = new List(description, index);
+  UI.addList(list);
+  store.addList(list);
+  window.location.reload();
+  UI.clearFields();
+});
+
+$('#to-do-lists').on('dblclick', 'li', function active() {
```

Comment:
- **[OPTIONAL]** I highly recommend avoiding the double-click event and using just a normal one-click event instead it will improve the user experience way more 👍🏼